src/Storages/System/StorageSystemServerSettings.cpp

```diff
@@ -0,0 +1,37 @@
+#include <Storages/System/StorageSystemServerSettings.h>
+#include <DataTypes/DataTypeString.h>
+#include <DataTypes/DataTypesNumber.h>
+#include <Interpreters/Context.h>
+#include <Core/ServerSettings.h>
+
+namespace DB
+{
+NamesAndTypesList StorageSystemServerSettings::getNamesAndTypes()
+{
+    return {
+        {"name", std::make_shared<DataTypeString>()},
+        {"value", std::make_shared<DataTypeString>()},
```

Comment:
Not sure but if the default value is available, it would be nice to include it. (same for system table `settings`)

Reply:
`system.settings` doesn't have a column with default values, but I can probably add it too.